packages/react/src/components/Theme/__tests__/Theme-test.js

```diff
@@ -90,4 +95,49 @@ describe('usePrefersDarkScheme', () => {
 
     expect(screen.getByTestId('default')).toHaveTextContent('dark');
   });
+
+  it('should call usePrefersDarkScheme', () => {
+    jest.resetModules();
+    usePrefersDarkScheme(); //mock the function
+
+    jest.spyOn(hooks, 'useMatchMedia').mockImplementation(() => false);
+    render(
+      <Theme theme="g10">
+        <DarkTestComponent id="default-dark" />
+      </Theme>
+    );
+
+    expect(screen.getByTestId('default-dark')).toHaveTextContent('light');
+    expect(hooks.useMatchMedia).toHaveBeenCalledWith(
+      '(prefers-color-scheme: dark)'
+    );
+  });
+});
+
+describe('GlobalTheme', () => {
+  jest.resetModules();
```

Comment:
This could move to be within the test body instead of this outer scope

Reply:
Hi @tay1orjones . Yes. I have moved that line `jest.resetModules()` inside the test body. Thank you